ce/common/config/rush/pnpm-lock.yaml

```diff
@@ -85,14 +85,14 @@ dependencies:
 packages:
 
   /@azure/abort-controller/1.1.0:
-    resolution: {integrity: sha512-TrRLIoSQVzfAJX9H1JeFjzAoDGcoK1IYX1UImfceTZpsyYfWr09Ss1aHW1y5TrrR3iq6RZLBwJ3E24uwPhwahw==}
+    resolution: {integrity: sha1-eI7nhFelWvihrTQqyxgjg9IRkkk=}
```

Comment:
Is this use of SHA1 against policy?

Reply:
I tried to get the sha512s to come back; that's why there's lockfileVersion being set here. Unfortunately it looks like this value comes from what is *advertised* by the feed when the package is published, and Azure DevOps feeds appear to advertise only the SHA1 value :(